tests/acceptance/requirements.txt

```diff
@@ -6,3 +6,4 @@ oslo.utils==1.7.0
 PyHamcrest>=1.8.3
 git+https://github.com/qaenablers/python-qautils.git@v1.2.0
 pika==0.10.0
+behave==1.2.5
```

Comment:
For the moment, we are considering the acceptance test project as an independent 'subproject' in the repo of the component. This subproject manages its own requirement.txt, installation procedure and documentation.